avatars/client.py

```diff
@@ -150,19 +150,19 @@ def request(
 
         if result.status_code != 200:
             json = result.json()
-            value = json.get('detail')
-            if result.status_code == 401 and isinstance(value, str) and "auth" in value :
+            value = json.get("detail")
+            if result.status_code == 401 and isinstance(value, str) and "auth" in value:
                 raise Exception("You are not authenticated.")
-
             standard_error = _get_nested_value(json, "message")
 
-            validation_error = _get_nested_value(json, "msg")
-            field = _get_nested_value(json, "loc")[-1]
-
             if standard_error:
                 error_msg = standard_error
-            elif validation_error:
-                error_msg = f"{validation_error}: {field}"
+            elif validation_error := _get_nested_value(json, "msg"):
```

Comment:
So we consider at least python3.9+ only is used on customer computer? is that true?

Reply:
Hmm, that implies a major version change then...